templates/admin/paywall-settings.php

```diff
@@ -120,10 +111,13 @@
                     <button type="button" onclick="document.getElementById('logout_button_text_color').value = '#fff';">Reset</button>
                 </td>
             </tr>
-
+            <!--NEW Advanced Settings tab-->
+            <tr>
+                <th colspan="2"><h2>Advanced Settings</h2></th>
+            </tr>
             <!--Blocklist Field -->
             <tr>
-                <th scope="row"><label for="paybutton_blocklist">Blocklisted eCash Addresses</label></th>
+                <th scope="row"><label for="paybutton_blocklist">Blacklisted eCash Addresses (optional)</label></th>
                 <td>
                     <textarea name="paybutton_blocklist" id="paybutton_blocklist" rows="4" cols="50"><?php
                         // Convert the blocklist array into a comma-separated string for display
```

Comment:
Nit but let's just update `blocklist` -> `blacklist` everywhere so it's consistent.